server.js

```diff
@@ -1,30 +1,131 @@
 import express from "express";
 import cors from "cors";
+import netflixData from "./data/netflix-titles.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+    const welcomePage = {
+      Hello:
+      "Hello there! Here you'll find an API with information about 1 375 Netflix movies and TV-shows!",
+      Routes: [{
+        "/content": "GET the whole array of movies and tv-shows",
+        "/content/id/'id-number": "GET a specific movie/tv-show by it's unique id-number",
+        "/content/title/'title of movie/show": "GET movies/tv-shows by title",
```

Comment:
I think this path should either be changed to content/'title of movie/show' as is written in the code on line 66, or the path in line 66 should be changed to /content/titles/'title of movie/show. I think the latter one might be good because then it's very obvious that it is a title that comes next :)